wiki-jpa-dao/src/main/java/org/exoplatform/wiki/jpa/JPADataStorage.java

```
@@ -592,6 +646,19 @@ public void addRelatedPage(Page page, Page relatedPage) throws WikiException {
     pageDAO.update(pageEntity);
   }
 
+  @Override
+  public List<Page> getPagesOfWiki(String wikiType, String wikiOwner) {
+    if (StringUtils.isBlank(wikiOwner) || StringUtils.isBlank(wikiType)) {
+      return Collections.emptyList();
```

We should return an exception here IMO, since empty values for these arguments are not allowed.

---

done